<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dialog</class>
 <widget class="QDialog" name="dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>458</width>
    <height>309</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>458</width>
    <height>309</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>458</width>
    <height>309</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MyFlashcards</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>441</width>
     <height>101</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="main_lbl">
      <property name="font">
       <font>
        <family>Bahnschrift</family>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Ho≈ügeldin!</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>441</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QLabel" name="secondary_lbl">
      <property name="font">
       <font>
        <family>Bahnschrift Light SemiCondensed</family>
        <pointsize>12</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLineEdit" name="edit_word">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>170</y>
     <width>381</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>230</y>
     <width>441</width>
     <height>61</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="btnGetWord">
      <property name="text">
       <string>Yeni Kelime</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnAnswer">
      <property name="text">
       <string>Cevapla</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnAdd">
      <property name="text">
       <string>Yeni Kelime Ekle</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
